x/staking/keeper/msg_server.go

```diff
@@ -177,14 +177,22 @@ func (k msgServer) EditValidator(goCtx context.Context, msg *types.MsgEditValida
 		validator.MinSelfDelegation = (*msg.MinSelfDelegation)
 	}
 
-	if msg.EthAddress != "" {
-		validator.EthAddress = msg.EthAddress
-	}
-
 	if msg.Orchestrator != "" {
+		_, err := k.validateOrchestratorAddress(ctx, msg.Orchestrator)
```

Comment:
Are you sure you want to error when a validator exists if you're trying to edit it?

Reply:
We don't want to have duplicate orchestrator or Ethereum address. If some other validatore has that address, we shouldnt continue. No?

Reply:
For inserting a new validator, yes. But editing a validator (not sure what that even means?) requires the validator to exist prior to being edited, no?

Reply:
We look first if the validatore address by its address. Then, we start checking if the new orchestrator address has been given to some other validatore. 
So, that one is looking for a validatore by orchestrator address. If any other validatore has that orchestrator address, we stop.
Or I am missing something

Reply:
I think this makes sense, we want to throw an error if someone is trying to update their vaildator with an invalid orchestrator address. If we don't, then we won't be able to verify their attestations, and while it wouldn't be protocol breaking, I think it would be a terrible UX, as they'd have to debug why they can't submit signatures.